Userland/Games/2048/Game.h

```diff
@@ -119,6 +119,7 @@ class Game final {
     u32 m_target_tile { 0 };
 
     bool m_evil_ai { false };
+    bool m_want_to_continue { false };
```

Comment:
This flag isn't reset even after restarting a game, so there won't be any more "winning" until the user relaunches the game.

Reply:
Yes, I missed that corner case.

Reply:
@alimpfard if you see the code of start `start_a_new_game`, a new game object is constructed there, so `m_want_to_continue` is automatically reset to `false`.